tasks/install_packages.yml

```diff
@@ -0,0 +1,9 @@
+- name: Install all required packages
+  apt:
+    name: "{{ item }}"
+  with_items:
+    - open-iscsi
+    - systemd-container
+    - sed
+    - openvswitch-common
```

Comment:
Shouldn't that be done by an ovs role?

Reply:
Fair enough, the should be nen option to select the default xen network configuration script and only install the packages accordingly. I completely forgot this task. Tanks.